Cannot find module 'webpack' upgrade from 0.28.0 to 0.28.1

Using in a github actions repo.

Below output from dependabot but occurring locally as well
https://github.com/tonyhallett/artifacts-url-comments/pull/8/checks?check_run_id=2287289635
```
> ncc build --source-map --license licenses.txt

ncc: Version 0.28.1
ncc: Compiling file index.js
HookWebpackError: Cannot find module 'webpack'
Require stack:
- /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js
- /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/cli.js
    at makeWebpackError (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:320193)
    at /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:144666
    at eval (eval at create (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117070), <anonymous>:13:1)
    at fn (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:116235)
    at _next1 (eval at create (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117070), <anonymous>:11:1)
    at eval (eval at create (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117070), <anonymous>:27:1)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
-- inner error --
Error: Cannot find module 'webpack'
Require stack:
- /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js
- /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at WebpackModuleFileIterator.resolve [as requireResolve] (internal/modules/cjs/helpers.js:94:19)
    at WebpackModuleFileIterator.getActualFilename (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951349)
    at WebpackModuleFileIterator.internalCallback (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951052)
    at WebpackModuleFileIterator.iterateFiles (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:950726)
    at /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:934018
    at WebpackChunkModuleIterator.iterateModules (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:944891)
    at PluginChunkReadHandler.processChunk (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:933969)
    at _loop_1 (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:947680)
    at WebpackCompilerHandler.iterateChunks (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:948545)
caused by plugins in Compilation.hooks.processAssets
Error: Cannot find module 'webpack'
Require stack:
- /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js
- /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at WebpackModuleFileIterator.resolve [as requireResolve] (internal/modules/cjs/helpers.js:94:19)
    at WebpackModuleFileIterator.getActualFilename (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951349)
    at WebpackModuleFileIterator.internalCallback (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951052)
    at WebpackModuleFileIterator.iterateFiles (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:950726)
    at /home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:934018
    at WebpackChunkModuleIterator.iterateModules (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:944891)
    at PluginChunkReadHandler.processChunk (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:933969)
    at _loop_1 (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:947680)
    at WebpackCompilerHandler.iterateChunks (/home/runner/work/artifacts-url-comments/artifacts-url-comments/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:948545)
```

I have the same problem:
`> yarn ncc build ./src/index.ts --minify --license licenses.txt`
```
ncc: Version 0.28.1
ncc: Compiling file index.js
ncc: Using typescript@4.2.3 (local user-provided)
HookWebpackError: Cannot find module 'webpack'
Require stack:
- /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js
- /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/cli.js
    at makeWebpackError (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:320193)
    at /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:144666
    at eval (eval at create (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117070), <anonymous>:11:1)
    at fn (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:116235)
    at _next0 (eval at create (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117070), <anonymous>:9:1)
    at eval (eval at create (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117070), <anonymous>:24:1)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
-- inner error --
Error: Cannot find module 'webpack'
Require stack:
- /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js
- /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at WebpackModuleFileIterator.resolve [as requireResolve] (internal/modules/cjs/helpers.js:80:19)
    at WebpackModuleFileIterator.getActualFilename (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951349)
    at WebpackModuleFileIterator.internalCallback (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951052)
    at WebpackModuleFileIterator.iterateFiles (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:950726)
    at /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:934018
    at WebpackChunkModuleIterator.iterateModules (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:944891)
    at PluginChunkReadHandler.processChunk (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:933969)
    at _loop_1 (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:947680)
    at WebpackCompilerHandler.iterateChunks (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:948545)
caused by plugins in Compilation.hooks.processAssets
Error: Cannot find module 'webpack'
Require stack:
- /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js
- /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at WebpackModuleFileIterator.resolve [as requireResolve] (internal/modules/cjs/helpers.js:80:19)
    at WebpackModuleFileIterator.getActualFilename (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951349)
    at WebpackModuleFileIterator.internalCallback (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:951052)
    at WebpackModuleFileIterator.iterateFiles (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:950726)
    at /Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:934018
    at WebpackChunkModuleIterator.iterateModules (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:944891)
    at PluginChunkReadHandler.processChunk (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:933969)
    at _loop_1 (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:947680)
    at WebpackCompilerHandler.iterateChunks (/Users/pataruco/development/continous-compliance/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:948545)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/.npmignore
+++ b/.npmignore
@@ -1,5 +1,0 @@
-*
-!scripts/cli.js
-!dist/**/*.js
-!dist/**/*.cache
-!dist/**/*.ts

--- a/package.json
+++ b/package.json
@@ -8,6 +8,9 @@
   "bin": {
     "ncc": "./dist/ncc/cli.js"
   },
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "build": "node scripts/build",
     "build-test-binary": "cd test/binary && node-gyp rebuild && cp build/Release/hello.node ../integration/hello.node",

